packages/blockchain-link/src/workers/solana/index.ts

```diff
@@ -142,12 +185,14 @@ const getAccountInfo = async (request: Request<MessageTypes.GetAccountInfo>) =>
     const { details = 'basic' } = payload;
     const api = await request.connect();
 
-    const publicKey = new PublicKey(payload.descriptor);
+    const publicKey = address(payload.descriptor);
```

Comment:
We call public keys what they really are now: addresses. Furthermore, in the new API, they are just strings.